src/views/Upload.vue

```diff
@@ -4,45 +4,48 @@
 			class="w-screen h-screen absolute bg-primary-gradient clip-half-circle"
 		/>
 		<Navigation class="z-10" />
-		<div class="container mx-auto rounded-lg shadow-xl my-12 bg-white z-10">
-			<div class="px-8 pt-10 pb-2">
-				<h1 class="text-3xl leading-10 font-bold text-gray-900">
-					Kontrola plagiátorstva
-				</h1>
-				<h3 class="text-xl mt-2 leading-8 text-gray-800">
-					Pridajte jednu alebo viac prác, ktoré si prajete
-					skontrolovať. Nahrajte súbory alebo vložte text práce.
-				</h3>
-			</div>
-			<div class="px-20 pb-10 pt-5">
-				<div class="flex">
-					<UploadTab
-						v-for="tab in tabs"
-						:key="tab"
-						:title="tab"
-						:active="tab === selectedTab"
-						@click.native="selectedTab = tab"
-					/>
+		<div class="flex items-center md:container md:mx-auto w-full">
+			<div class="mx-0 h-screen md:h-auto md:mx-auto md:rounded-lg shadow-xl mt-4 md:my-12 bg-white z-10">
+				<div class="px-6 md:px-8 pt-4 md:pt-10 pb-1 md:pb-2">
+					<h1 class="text-3xl leading-normal md:leading-10 font-bold text-gray-900">
+						Kontrola plagiátorstva
+					</h1>
+					<h3 class="text-xl mt-2 leading-9 md:leading-8 text-gray-800">
+						Pridajte jednu alebo viac prác, ktoré si prajete
```

Comment:
The text is so large here.
![image](https://user-images.githubusercontent.com/27869979/109662896-d3cbd480-7b6b-11eb-9c1d-c9e63008fcac.png)

Reply:
Doesn't look good.
![image](https://user-images.githubusercontent.com/27869979/109663011-ee05b280-7b6b-11eb-9278-5054c7852c96.png)